SharedProjects/BabylonExport.Entities/BabylonPBRClearCoat.cs

```diff
@@ -13,6 +13,8 @@ public class BabylonPBRClearCoat
 
         [DataMember]
         public float roughness { get; set; } = 0.0f;
+        [DataMember]
+        public BabylonTexture textureRoughness { get; set; }
```

Comment:
to be consistent with tintTexture
```suggestion
        public BabylonTexture roughnessTexture { get; set; }
```

Reply:
This name is exact mapping of Babylon JS class, which is itself not consistent.